packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -189,8 +189,10 @@ export const Combobox = <T extends string | number>({
         items.findIndex((opt) => opt.label === inputValue || opt.value === inputValue) === -1
           ? {
               // Type casting needed to make this work when T is a number
-              value: inputValue as unknown as T,
+              value: (t('combobox.custom-value.label', 'Custom value: ') + inputValue) as unknown as T,
```

Comment:
You need to add the `Custom value:` to the label property instead. Currently, this will set the actual selected value to include `Custom value: `
```suggestion
              value: inputValue as unknown as T,
              label: t('combobox.custom-value.label', 'Custom value: ') + inputValue,
```